source/languages/en/dataplatform/using-dataplatform/configuration/replace-spark-cluster-manager.md

```diff
@@ -164,7 +166,7 @@ You can clean up the index.html file when you are done by running: `sudo rm inde
 
 Make sure the README.md is in pwd. Then enter:
 
-```
+```Spark-Shell
```

Comment:
Why `Spark-Shell`? This looks a lot like JavaScript to me, should we use that highlighting?